builds/misc/packages-release.yaml

```diff
@@ -435,5 +435,161 @@ stages:
             BASE_VERSION=`cat $BUILD_SOURCESDIRECTORY/edgelet/version.txt`
             $(Build.SourcesDirectory)/scripts/linux/publishReleasePackages.sh -p $(os) -w $(System.ArtifactsDirectory) -d $(System.ArtifactsDirectory)/$(artifactName) -v "$BASE_VERSION" -s "github.com" -b $(Build.SourceBranch)
           env:
-            GITHUB_PAT: "$(GitHubAccessToken)"    
-          name: publish_artifacts 
+            GITHUB_PAT: "$(GitHubAccessToken)"
+          name: publish_artifacts
+
+##############################################################################
+  - stage: smokeTests
+##############################################################################
+    displayName: Post-release Smoke Tests
+    dependsOn:
+      - PublishPackagesMicrosoft
+      - PublishPackagesGithub
+    jobs:
+    - job: smokeTests
+      displayName: Release Artifact Smoke Tests
+      strategy:
+        matrix:
+          Ubuntu1804-amd64:
+            pool_name: 'Azure-IoT-Edge-1ES-Hosted-Linux'
+            agent_demands: 'ImageOverride -equals agent-aziotedge-ubuntu-18.04-docker'
```

Comment:
I'd prefer 20.04

Reply:
Nope. I'm not gonna deal with the shady 1ES focal OS linux repository right now. The apt source for Focal has part Bionic as a source repo and part Focal for the higher level apps. This messes up when we tried to pull `iotedge` from the default repo. I originally have Focal & Bionic & Deb working but 1ES decided to update some of their PMC source and doing more questionable things. I'm doing a mitigation here by using the most known stable that is Bionic.